### PR TITLE
Add APIs dateCellContentRender&monthCellContentRender

### PR DESCRIPTION
**Pull request by wuhaixing**
Add APIs dateCellContentRender&monthCellContentRender in FullCalendar for custom content render in origin cell,but not replace it.

### CODE REVIEW

**Comment by benjycui** [comment]
Please `npm run lint` and fix those ESLint errors, thanks.

**Comment by wuhaixing** [comment]
`onSelect` test failed!I'm not dig into it yet,but I think the problem is month cell element is an `<a/>`,and I wrapped it with a `<div/>`,because I want to show some events in cell.

**Comment by yiminghe** [comment]
I don't see a need for an additional prop.

Using dateCellRender to provide your own cell is enough

**Comment by wuhaixing** [comment]
Yes,it's enough.But what I need is just to **append** something to cell,not **replace** it.Such as todos,courses.`dateCellRender` is enough to add these events to full-calendar,but I need rewrite something like `getIdFromDate(current)`,`dateClass` things whats the component already done.
        
        if (dateRender) {
          dateHtml = dateRender(current, value);
        } else {
          dateHtml = (
            <span
              key={getIdFromDate(current)}
              className={dateClass}
              aria-selected={selected}
              aria-disabled={disabled}
            >
              {current.getDayOfMonth()}
            </span>);
        }

File: src/month/MonthTable.js [hunk 1]
```diff
@@ -90,7 +90,18 @@ class MonthTable extends Component {
...
-          cellEl = <a className={`${prefixCls}-month`}>{monthData.content}</a>;
+          let cellContent;
+          if (props.contentRender) {
+            const currentValue = value.clone();
+            currentValue.rollSetMonth(monthData.value);
+            cellContent = props.contentRender(currentValue, locale);
+          }
+          cellEl = (
+                   <div>
+                    <a className={`${prefixCls}-month`}>{monthData.content}</a>
```
Comment (yiminghe) [inline src/month/MonthTable.js]: 替换掉 monthData.content ，和 dateCellContentRender 保持一致

File: src/date/DateTBody.jsx [hunk 2]
```diff
@@ -187,6 +191,7 @@ const DateTBody = React.createClass({
               aria-disabled={disabled}
             >
               {current.getDayOfMonth()}
```
Comment (yiminghe) [inline src/date/DateTBody.jsx]: 不要append，直接提换掉

Comment (wuhaixing) [inline src/date/DateTBody.jsx]: 我比较倾向于写成下面这样，放到各自的标签中，但还没来得及看className的命名规则。

         <div
              key={getIdFromDate(current)}
              className={dateClass}
              aria-selected={selected}
              aria-disabled={disabled}
            >
              <span className="fc-time">{current.getDayOfMonth()}</span>
              <div className="fc-content"> {dateContent}</div>
            </div>)

Comment (yiminghe) [inline src/date/DateTBody.jsx]: content ， 东西要换就全部换掉

File: src/date/DateTBody.jsx [hunk 3]
```diff
@@ -179,15 +179,32 @@ const DateTBody = React.createClass({
...
+                className={dateClass}
+                aria-selected={selected}
+                aria-disabled={disabled}
+              >
+                <span className={`${prefixCls}-date-value`}>
+                  {current.getDayOfMonth()}
+                </span>
+                <div className={`${prefixCls}-date-content`}>
+                  {props.contentRender(current, value)}
+                </div>
```
Comment (yiminghe) [inline src/date/DateTBody.jsx]: 199-204 换成 {props.contentRender(current, value)}

**Pull request merged**